### DISCUSSION TITLE
OpenDTU compatible with Solenso SOL-H800

### DISCUSSION THREAD

**Discussion by mathieucarbou** [tbnobody/OpenDTU#731] [General]
Hello,

Good news,

I got the confirmation today that OpenDTU also works with Solenso SOL-H800.
And I know several people now using it with Solenso SOL-H400 too (but this compatibility was already confirmed).

Best regards,

Mathieu.

**Comment by mathieucarbou** [comment]
@tbnobody FYI

![markup_1931022108](https://user-images.githubusercontent.com/61346/227317744-60abd919-a405-4deb-be00-375c15904878.png)
![markup_1210611915](https://user-images.githubusercontent.com/61346/227317748-617c2155-6ca6-483b-841b-bfb0ab8ccc7e.png)

**Reply by tbnobody** [reply]
Thank you! Will add it to the Readme

**Reply by huckes** [reply]
Hi, there seem to be two types of Solenso converts: SOLHnnn and SOL-Hnnn**H**, pls. refer to https://www.solenso-global.com/en/h-col-203.html. The latter one looks like a copy or re-named Hoymiles converter. Are both types compatibe or only the latter?

**Reply by mathieucarbou** [reply]
> Hi, there seem to be two types of Solenso converts: SOLHnnn and SOL-Hnnn**H**, pls. refer to https://www.solenso-global.com/en/h-col-203.html. The latter one looks like a copy or re-named Hoymiles converter. Are both types compatibe or only the latter?

I know for sure OpenDTU works with the `SOL-H400` and `SOL-H800`. I am pretty sure that it also works with the `-H` version.

You could confirm with the company itself, but it seems to me that the H and non-H versions are the same except for the connector to daisy-chain the inverters. Also, The -H version weirdly like you said looks a bit too close to the Hymiles ones lol (maybe why the H at the end lol ?)

**Reply by stephan-cardot** [reply]
Hello,
The H version seems to be more linked to the physical connector.
Do you know if version SOLH1000H is also supported by OpenDTU?

**Reply by mathieucarbou** [reply]
IMO, yes it works.  Here is my logic behind that:

Hoymiles has 2 lines:
- `HM-*` : work on 2.4Ghz with a nRF24L01 radio card
- `HMS-*` : work on Sub-1G, with another type of radio card (CMT) which is currently harder to find (only Ebyte and a few other sellers are selling it and it has to be soldered: there is not yet variations with antenna connectors soldered and Dupont connectors easy to use with an ESP like the nRF24L01 variants).

The `HMS-*` line is pretty new.
So logically, all `SOL-H*` should be like the Hoymiles `HM-*`, like it was confirmed with the 350, 400 and 800.
If Solenso were to sell some HMS-* variants branded with their make, it owuld probably be another line.

In any case, if you can confirm that it works, please report to @tbnobody to add it to the list of supported hardware.

**Reply by stephan-cardot** [reply]
Hello,
I've build an OpenDTU to monitor my Solenso Sol-H1000H inverter but I don't know how to enter the serial number of my inverters because they contain letters? Example : A11002PL3-25A

**Reply by avandorp** [reply]
Solenso's manual shows a barcode on the inverter with a number to be entered in the dtu software. Does yours have anything like that directly on the inverter?

**Reply by stephan-cardot** [reply]
Hello,
No number with the barcode. Below the sticker on one of my inverters:
<img width="227" alt="image" src="https://github.com/tbnobody/OpenDTU/assets/65170595/831b8ffd-e680-48ce-b54e-96788760ced6">

**Comment by heinihuber** [comment]
I have already bought a (by OpenDTU-OnBattery) non-supported inverter once and am now looking for a reasonable alternative. As there are users in this article who have bought an inverter from Solenso, I would be delighted if someone could take pity on me and answer my question!

Has the Solenso SOL-H800 Wi-Fi support? If yes, could the AP mode be disabled after integration to the existing Wi-Fi is done?
